tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1083,7 +1083,14 @@ Duration <DURATION>"
 `;
 
 exports[`container_types.test.w 1`] = `
-"error: Unknown parser error
+"error: Unexpected 'number'
+  --> ../../../examples/tests/invalid/container_types.test.w:3:24
+  |
+3 | let arr2 = Array<num> {1, 2, 3};
+  |                        ^ Unexpected 'number'
```

Comment:
The snapshots still look off in a few places.

In this case, I think we can remove this since it's specifically related to the {} syntax.

Reply:
Oh, I thought snapshots got updated automatically. I'll do it